lib/main.coffee

```diff
@@ -144,6 +144,17 @@ module.exports =
   provideLinter: ->
     helpers = require('atom-linter')
     path = require('path')
```

Comment:
To make stuff simpler, we could make this `Path` and then use `path` to store path value below